### DISCUSSION TITLE
How to get number of open and closed issue of a repo

### DISCUSSION THREAD

**Discussion by dejokz** [PyGithub/PyGithub#1848] [Q&A]
is there a way to get the number of open and closed issue from a given repo.
I'm aware 
```
repo = g.get_repo("PyGithub/PyGithub")
open_issues = repo.get_issues(state='open')
```

and using for loop i can go through the paginatedList. But what about the closed issues?
again i just need the number of open and closed, i'm not too concerened with what those issues are.

**Comment by walterrowe** [comment]
Can you use len(open_issues), and state=closed for closed?
